components/Starknet/modules/ROOT/pages/faq.adoc

```diff
@@ -0,0 +1,18 @@
+[id="FAQ"]
+= Frequently asked questions
+
+== What does the Sequencer do with the gas fees? (ETH or STRK)
+
+Sequencers have the role of converting STRK fees received from users into ETH to meet operational expenses. However, the frequency of these conversions can vary depending on the financial management approach of each sequencer's operator.
+At present, the sequencer employs an ETH/STRK oracle to ascertain the appropriate number of STRK tokens to be charged as fees to its users.
+
+For more information on fees and how they are calculated, see xref:architecture_and_concepts:Network_Architecture/fee-mechanism.adoc[].
+
+== What are rebates used for?
+
+Rebates in Starknet token design are used as a way to distribute Starknet’s native token (STRK) to eligible users, builders, and community members who have contributed to Starknet. These rebates will enable the community to pay fees in STRK, and engage in the decentralization of the network by participating in governance votes and in the decentralized proof-of-stake (PoS) model that will be implemented in later stages. The rebates will be distributed according to the Provisions blog post, which details the purpose of Provisions, the guiding principles for its distribution, and the criteria for its allocation.
+For more information, see link:https://medium.com/starkware/part-3-starknet-token-design-5cc17af066c6[Starknet Token Design] on Medium.
+
+== Is there any burning mechanism?
+
+StarkNet handles excess tokens from transaction fees by allowing sequencers to convert the fees paid in STRK and sell them for ETH to cover operating costs and not through a burning mechanism. The operator of every sequencer chooses how to manage their finances, so they may convert frequently or infrequently. Currently, the sequencer uses an ETH/STRK oracle to determine how many STRK tokens to charge its users as a fee.
```

Comment:
```suggestion
Starknet handles excess tokens from transaction fees by enabling sequencers to convert the fees paid in STRK to ETH to cover operating costs, rather than through a burning mechanism. The operator of every sequencer chooses how to manage their finances, so they determine how frequently to convert. Currently, the sequencer uses an ETH/STRK oracle to determine how many STRK tokens to charge its users as a fee.
```
I like the word _determine_ here more than _ascertain_ in line 7. Can you change that to be consistent?

Also, avoid ambiguous words. My anti-favorites is _may_, because it means either _might_ or _can_.